src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,113 @@
+            FOLLOWER_ELEVATOR_MOTOR_ID = 3,
+            ELEVATOR_MAG_ENCODER_ID = 4,
+            ANGLE_CANCODER_ID = 5;
+
+    private static final double
+            ELEVATOR_MAG_ENCODER_OFFSET = 0.0,
+            ANGLE_CANCODER_OFFSET = 0.0;
+
+    private static final boolean
+            MASTER_ANGLE_INVERTED_VALUE = false,
+            FOLLOWER_ANGLE_INVERTED_VALUE = false,
+            MASTER_ELEVATOR_INVERTED_VALUE = false,
+            FOLLOWER_ELEVATOR_INVERTED_VALUE = true,
+            ELEVATOR_MAG_ENCODER_PHASE_VALUE = true;
+    private static final SensorDirectionValue ANGLE_CANCODER_SENSOR_DIRECTION = SensorDirectionValue.Clockwise_Positive;
+
+    private static final double
+            P = 1.32,
+            I = 3.1,
+            D = 0;
+    private static final PIDController PID_CONTROLLER = new PIDController(P, I, D);
+
+    private static final CANSparkMax.IdleMode
+            ANGLE_NEUTRAL_MODE_VALUE = CANSparkMax.IdleMode.kBrake,
+            ELEVATOR_NEUTRAL_MODE_VALUE = CANSparkMax.IdleMode.kBrake;
+    private static final AbsoluteSensorRangeValue ANGLE_ENCODER_SENSOR_RANGE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+
+    private static final CANSparkMax
+            MASTER_ANGLE_MOTOR = new CANSparkMax(MASTER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ANGLE_MOTOR = new CANSparkMax(FOLLOWER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            MASTER_ELEVATOR_MOTOR = new CANSparkMax(MASTER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ELEVATOR_MOTOR = new CANSparkMax(FOLLOWER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private static final TalonSRX ELEVATOR_MAG_ENCODER = new TalonSRX(ELEVATOR_MAG_ENCODER_ID);
+    private static final CANcoder ANGLE_CANCODER = new CANcoder(ANGLE_CANCODER_ID);
+
+    static {
+        motors_angle_config();
+        motors_elevator_config();
+        configureElevatorEncoder();
+    }
+
+    private static void motors_elevator_config() {
+        MASTER_ELEVATOR_MOTOR.restoreFactoryDefaults();
+        FOLLOWER_ELEVATOR_MOTOR.restoreFactoryDefaults();
+
+        MASTER_ELEVATOR_MOTOR.setInverted(MASTER_ELEVATOR_INVERTED_VALUE);
+        FOLLOWER_ELEVATOR_MOTOR.setInverted(FOLLOWER_ELEVATOR_INVERTED_VALUE);
+
+        MASTER_ELEVATOR_MOTOR.setIdleMode(ANGLE_NEUTRAL_MODE_VALUE);
+        FOLLOWER_ELEVATOR_MOTOR.setIdleMode(ANGLE_NEUTRAL_MODE_VALUE);
+
+        ELEVATOR_MAG_ENCODER.setSelectedSensorPosition(ELEVATOR_MAG_ENCODER.getSelectedSensorPosition() - ELEVATOR_MAG_ENCODER_OFFSET);
+        ELEVATOR_MAG_ENCODER.setSensorPhase(ELEVATOR_MAG_ENCODER_PHASE_VALUE);
+
+    }
+
+    private static void configureElevatorEncoder() {
+        ELEVATOR_MAG_ENCODER.setSelectedSensorPosition(ELEVATOR_MAG_ENCODER.getSelectedSensorPosition() - ELEVATOR_MAG_ENCODER_OFFSET);
+        ELEVATOR_MAG_ENCODER.setSensorPhase(ELEVATOR_MAG_ENCODER_PHASE_VALUE);
+    }
```

Comment:
Restore factory defaults and [select the remote encoder](https://v5.docs.ctr-electronics.com/en/stable/ch14_MCSensor.html?highlight=configselected#software-select-sensor) (absolute)